### DISCUSSION TITLE
5.2 Release Date

### DISCUSSION THREAD

**Discussion by sanderkooij** [dotnet/SqlClient#2244] [General]
Is there any information available on when 5.2 will be released? We would like to upgrade to .Net 8.0, but we are not able because of this issue in the SqlClient: [https://github.com/dotnet/SqlClient/issues/1930](https://github.com/dotnet/SqlClient/issues/1930)
When using the 5.2 pre-release version everything seems to works fine! Unfortunately, our company does not allow the usage of prerelease packages, so we are unable to upgrade to .Net 8.0 at this time. 
Any insight on a possible release date will be highly appreciated!

**Comment by ErikEJ** [comment]
January 2024 has been mentioned.

**Reply by sanderkooij** [reply]
Thank you for the quick reply

**Comment by David-Engel** [comment]
February is the current target. https://github.com/dotnet/SqlClient/milestones
